documentation/modules/appendix_crds.adoc

```diff
@@ -2931,6 +2931,8 @@ Used in: xref:type-KafkaRebalance-{context}[`KafkaRebalance`]
 |string array
 |skipHardGoalCheck                        1.2+<.<|Whether to allow the hard goals specified in the Kafka CR to be skipped in rebalance proposal generation. This can be useful when some of those hard goals are preventing a balance solution being found. Default is false.
```

Comment:
Perhaps you could update "rebalance proposal" to "optimization proposal" throughout the Cruise Control API documentation?